bgpd/bgp_route.c

```diff
@@ -7902,15 +7882,9 @@ DEFUN (show_ip_bgp_ipv4,
     afi = strmatch(argv[idx]->text, "ipv6") ? AFI_IP6 : AFI_IP;
     if (argv_find (argv, argc, "unicast", &idx) || argv_find (argv, argc, "multicast", &idx))
       safi = bgp_vty_safi_from_arg (argv[idx]->text);
+    else if (argv_find (argv, argc, "encap", &idx) || argv_find (argv, argc, "vpn", &idx))
```

Comment:
This should be changed to use bgp_vty_find_and_parse_afi_safi_vrf (PR #107) 
or use the pattern:

  if (argv_find_and_parse_afi (argv, argc, &idx, &afi))
      argv_find_and_parse_safi (argv, argc, &idx, &safi);